scanpy/plotting/_baseplot_class.py

```diff
@@ -558,42 +559,28 @@ def _mainplot(self, ax):
 
         return normalize
 
-    def show(
-        self, show: Optional[bool] = None, save: Union[str, bool, None] = None,
-    ):
+    def render(self):
```

Comment:
I’m not sure `render` is the right name, since rendering doesn’t actually happen until you show or save the figure. Maybe `make_figure`?

Reply:
no problem to change this. As for the method being public.. if you pass the `ax` parameter you want to make the image but not call `plt.show`.  Thus the method should be public.

Reply:
Alright, sounds good